features_basic_operations.tex

```diff
@@ -0,0 +1,153 @@
+
+\subsubsection{Assumptions system}
+
+% Including the assumptions system
+
+
+SymPy has two assumptions systems, referred to as new-style and old-style assumptions.
+
+In the old-style assumptions system prepositions are assigned to symbols
+upon class construction, for example, to declare the symbol $i$ as positive integer,
+one would call
+
+\begin{verbatim}
+i = Symbol("i", integer=True, positive=True)
+\end{verbatim}
+
+querying the assumptions is handled through class methods
```

Comment:
I would call these "attributes". "Class method" has a specific meaning in Python that doesn't apply here.